server/routes/contact.py

```
@@ -0,0 +1,98 @@
+
+
+def send_emails_in_background(data):
+    """Background function to send emails"""
+    try:
+        # Send admin notification
+        admin_email = os.getenv("ADMIN_EMAIL", SMTP_USER)
+        send_contact_email(admin_email, 'admin', data)
+
+        # Send user confirmation
+        send_contact_email(data['email'], 'client', data)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)